packages/react-native/src/types/NotificationIOS.ts

```diff
@@ -125,21 +125,38 @@ export interface IOSForegroundPresentationOptions {
   /**
    * App in foreground dialog box which indicates when a decision has to be made
    *
-   * Defaults to false
+   * Defaults to true
+   * @deprecated Deprecated in iOS 14. Use `banner` and `list` instead
    */
   alert?: boolean;
+
   /**
    * App in foreground notification sound
    *
-   * Defaults to false
+   * Defaults to true
    */
   sound?: boolean;
+
```

Comment:
It seems we are falling back parameters to `true` in `packages/react-native/src/validators/validateIOSNotification.ts`.